easybuild/easyconfigs/h/HISAT2/HISAT2-2.1.0-intel-2017a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'HISAT2'
+version = '2.1.0'
+
+homepage = 'https://ccb.jhu.edu/software/hisat2/index.shtml'
+description = """HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads
+ (both DNA and RNA) against the general human population (as well as against a single reference genome)."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['ftp://ftp.ccb.jhu.edu/pub/infphilo/%(namelower)s/downloads/']
+sources = ['%(namelower)s-%(version)s-source.zip']
+patches = ['HISAT2-%(version)s_intel-fixes.patch']
+checksums = [
+    '89a276eed1fc07414b1601947bc9466bdeb50e8f148ad42074186fe39a1ee781',  # hisat2-2.1.0-source.zip
+    'd74a8ee4cc706b7ec1d9f48ca393fa2233f9d9f7988bd925be004532754a6174',  # HISAT2-2.1.0_intel-fixes.patch
```

Comment:
@smoors checksum should now be `7986dbcdb56b4bf5494dd70b215e90e321ca887dbdafc48de7a13e6b5abd038b` after adding the description/author?

Reply:
fixed.